cartographer_ros/cartographer_ros/node_options.cc

```diff
@@ -40,6 +40,10 @@ NodeOptions CreateNodeOptions(
       lua_parameter_dictionary->GetDouble("pose_publish_period_sec");
   options.trajectory_publish_period_sec =
       lua_parameter_dictionary->GetDouble("trajectory_publish_period_sec");
+  if (lua_parameter_dictionary->HasKey("use_pose_extrapolator")) {
```

Comment:
Add use_pose_extrapolator = true; to lua files to make sure users are aware of this new feature?